packages/php/email-editor/package.json

```diff
@@ -7,19 +7,48 @@
 		"lint:fix": "pnpm --if-present '/^lint:fix:lang:.*$/'",
 		"lint:fix:lang:php": "composer run-script phpcbf",
 		"lint:lang:php": "composer run-script phpcs",
-		"postinstall": "XDEBUG_MODE=off composer install --quiet"
+		"postinstall": "XDEBUG_MODE=off composer install --quiet",
+		"env:test": "pnpm env:dev",
+		"env:dev": "pnpm wp-env start --update",
+		"test:unit": "composer run-script test:unit",
+		"test:php:env": "composer run-script test:unit",
+		"test:integration": "composer run-script test:integration"
 	},
 	"license": "GPL-3.0-or-later",
-	"dependencies": {},
 	"devDependencies": {
+		"@wordpress/env": "10.17.0",
 		"rimraf": "5.0.5"
 	},
 	"config": {
 		"ci": {
 			"lint": {
 				"command": "lint",
 				"changes": "src/**/*.{php}"
-			}
+			},
+			"tests": [
+				{
+					"name": "PHP: 8.1 WP: latest",
+					"testType": "unit:php",
+					"command": "test:php:env",
```

Comment:
A bit misleading, how about using `test:php:ci` here which runs both `test:unit` and `test:integration`? This way we offset heavy environment spinoff in CI a b little bit.

Reply:
@kalessil Did you mean something such as I added in this commit: https://github.com/woocommerce/woocommerce/pull/56259/commits/5aa714f4a115bda27acfdfdfc473529843136796

Reply:
@lysyjan : yep, that's exactly it. Thanks for updating!